news/2023/2023-10-23-beatmap-spotlights-season-10-autumn-2023.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: "Beatmap Spotlights Season 10: Autumn 2023"
+date: 2023-10-23 18:00:00 +0000
+---
+
+You heard it right, the Beatmap Spotlights are back with the **10th Season**! As moody as autumn days can get, wake up and compete on the best beatmapping creations of last months right now!
+
+![](/wiki/shared/news/banners/beatmap-spotlights.jpg)
+
+After a bit of a breathing break to sort some of management surrounding Beatmap Spotlights, we'd like to announce that we are back with the **10th season of Beatmap Spotlights**. With Autumn 2023 season starting, we'd like to thank you for staying with us for 2-digit amount of iterations over the years of playlist-oriented history of Beatmap Spotlights! Even though this season will remain rather regular, we are preparing some bigger-schema changes for the next ones so stay prepared for that!
+
+If you just can't wait to try everything out, **the Beatmap Spotlights lobbies are now open for play NOW on osu!(lazer)**, so go get involved and start setting scores.
+
+If you still don't have any idea about what the entire Beatmap Spotlights thing could be, just read on below to find out how to jump into the world of playlist-oriented competition now:
+
+## How do I participate in the Beatmap Spotlights?
+
+To participate in the Beatmap Spotlights, download the latest version of the osu!(lazer) client from its [GitHub releases page](https://github.com/ppy/osu/releases/latest).
```

Comment:
```suggestion
To participate in the Beatmap Spotlights, go grab yourself the latest version of the osu!(lazer) client from this [download page](https://osu.ppy.sh/home/download).
```

feels friendlier to link the official download page rather than github

Reply:
yeah, probably a leftover from when it wasn't a thing on web